lib/delivery-locations-resolver.js

```diff
@@ -236,7 +236,13 @@ class DeliveryLocationsResolver {
     // location is not requestable:
     let deliveryLocation
     let eddRequestable
-    if (!isItemNyplOwned(item) || this.requestableBasedOnHoldingLocation(item)) {
+    const hasRecapCustomerCode = item.recapCustomerCode && item.recapCustomerCode[0]
+    const nyplItem = isItemNyplOwned(item)
+    if (!hasRecapCustomerCode) {
+      const requestableBasedOnHoldingLocation = nyplItem ? this.requestableBasedOnHoldingLocation(item) : true
+      deliveryLocation = requestableBasedOnHoldingLocation ? [''] : []
```

Comment:
Can you add a comment here explaining the purpose of the empty string array. I imagine it is to mock delivery locations length so the item maintains deliverability?
Will this require filtering out that empty string at some point down the line?